test/cli/test_kbsecret.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'test_helper'
+
+class CLITest < Minitest::Test
+  include Aruba::Api
+  include Helpers
+
+  def setup
+    setup_aruba
+  end
+
+  def test_help
+    run_command_and_stop "kbsecret help"
+    assert_match(/Usage:/, last_command_started.output.chomp)
+  end
+
+  def test_version
+    version_string = "kbsecret version #{KBSecret::VERSION}."
+    run_command_and_stop "kbsecret version"
+    assert_equal version_string, last_command_started.output.chomp
+  end
+
+  def test_commands
+    run_command_and_stop "kbsecret commands"
+
+    # NOTE: fragile, but no apparent means of getting a list of commands programmatically
```

Comment:
I think just the `run_command_and_stop` is sufficient for this test -- no need to do any assertions, as they'll be too fragile.

Reply:
fixed. if at some point you find need to expose the list of commands via a method, we could always add the assertions back in

Reply:
Sounds like a plan 👍